src/libxively/xi_backoff_status_api.c

```diff
@@ -94,8 +94,8 @@ void xi_cancel_backoff_event()
 #ifdef XI_BACKOFF_GODMODE
 void xi_reset_backoff_penalty()
 {
-    xi_globals.backoff_status.backoff_lut_i = 0;
-    xi_globals.backoff_status.decay_lut_i   = 0;
+    xi_globals.backoff_status.backoff_lut = 0;
+    xi_globals.backoff_status.decay_lut   = 0;
```

Comment:
hm. not sure about this. So here is the structure:
https://github.com/xively/xively-client-c/blob/master/src/libxively/xi_backoff_status.h
The `backoff_lut` and `decay_lut` are pointers on LUTs. The suffix `_i` is the current index on a certain LUT.
Doesn't the `xi_reset_backoff_penalty` function resets "only" the indices to 0? I don't think this reset function should just release the pointers on these tables. I guess the `_i` can stay for the backoff_lut. But yes, there is no `decay_lut_i`. Where it has gone, I don't know.

Reply:
Could you check where did the decay_lut_i go? Did it exist at all?

Reply:
Oh it gets even worse.  Nothing in our code even calls xi_reset_backoff_penalty

Reply:
Oh I see, it was meant for the application to be able to zero it out.  Interesting.  I'll look at it more.

Reply:
Good catch and my bad for not thinking things through.